Source/AlchemyVisionV1/AlchemyVision.swift

```diff
@@ -160,6 +162,24 @@ public class AlchemyVision {
         failure: (NSError -> Void)? = nil,
         success: ImageLink -> Void)
     {
+        // encode html document
+        guard let htmlEncoded = html.stringByAddingPercentEncodingWithAllowedCharacters(unreservedCharacters) else {
+            let failureReason = "Failed to percent encode HTML document."
```

Comment:
@Hsaylor: I added a helper function, `getImage(html:url:failure:success:)` that accepts an `NSURL` for the `html` parameter. I also added a couple of tests to make sure it's working.

Can you take another quick look and then merge the PR if you think it's ready? Thanks!